src/e2e/deploy.e2e.ts

```diff
@@ -0,0 +1,66 @@
+import { run } from "./utils/shell";
+import { DeployDocumentStoreCommand, DeployTokenRegistryCommand } from "../commands/deploy/deploy.types";
+import { isAddress } from "web3-utils";
+import { defaultRunParameters, EndStatus, owner } from "./utils/constants";
+import { generateDeployDocumentStoreCommand, generateDeployTokenRegistryCommand } from "./utils/commands";
+import { getSigner, retrieveTokenInfo, rolesCheck } from "./utils/contract-checks";
+import { checkTokenRegistrySuccess, defaultTokenRegistry } from "./utils/helpers";
+
+export const deployTokenRegistry = async (): Promise<void> => {
+  //should be able to deploy token-registry"
+  {
+    const tokenRegistryParameter: DeployTokenRegistryCommand = {
+      registryName: "Test Token",
+      registrySymbol: "TKN",
+      ...defaultTokenRegistry,
+    };
+
+    const command = generateDeployTokenRegistryCommand(tokenRegistryParameter, owner.privateKey);
+    const results = run(command);
+    const tokenRegistryAddress = checkTokenRegistrySuccess(results);
+    const signer = getSigner(defaultRunParameters.network, owner.privateKey);
+    const tokenInfo = await retrieveTokenInfo(signer, tokenRegistryAddress);
+    if (!(tokenInfo.name === tokenRegistryParameter.registryName)) {
+      throw new Error("tokenInfo.name === tokenRegistryParameter.registryName");
+    }
+    if (!(tokenInfo.symbol === tokenRegistryParameter.registrySymbol)) {
+      throw new Error("tokenInfo.symbol === tokenRegistryParameter.registrySymbol");
+    }
```

Comment:
y do we need to check for the name and symbol, when those are the parameters you specified?